system/security/shared/SecureUser.hpp

```diff
@@ -200,7 +200,7 @@ class CSecureUser : public CInterface,
 
     virtual CDateTime & getPasswordExpiration(CDateTime& expirationDate){ return expirationDate; }
     virtual bool setPasswordExpiration(CDateTime& expirationDate) { return true; }
-    virtual int getPasswordDaysRemaining() {return -1;}
+    virtual int getPasswordDaysRemaining() {return -2;}//never expires
```

Comment:
The constants should be used in espprotocol and the other places that call this function...